openlibrary/templates/lib/nav_foot.html

```diff
@@ -51,12 +51,12 @@ <h2>Help</h2>
     <div id="footer-details">
       <img id="archive-logo" src="/static/images/pantheon.png" alt="" aria-hidden="true">
       <div id="legal-details" >
-        <span>Open Library is an initiative of the <a href="//archive.org/">Internet Archive</a>, a 501(c)(3) non-profit, building a digital library of Internet sites and other cultural artifacts in  digital form. Other <a href="//archive.org/projects/">projects</a> include the <a href="//archive.org/web/">Wayback Machine</a>, <a href="//archive.org/">archive.org</a> and <a href="//archive-it.org">archive-it.org</a></span>
+        <span>$:_('Open Library is an initiative of the <a href="//archive.org/">Internet Archive</a>, a 501(c)(3) non-profit, building a digital library of Internet sites and other cultural artifacts in  digital form. Other <a href="//archive.org/projects/">projects</a> include the <a href="//archive.org/web/">Wayback Machine</a>, <a href="//archive.org/">archive.org</a> and <a href="//archive-it.org">archive-it.org</a>')</span>
       </div>
       $ git_rev_hash = get_git_revision_short_hash()
       $if git_rev_hash:
         <div id="version-details">
-          <span class="version">version <a href="https://github.com/internetarchive/openlibrary/commit/$(git_rev_hash)">$git_rev_hash</a></span>
+          <span class="version">$:_('version <a href="https://github.com/internetarchive/openlibrary/commit/$(git_rev_hash)">$git_rev_hash</a>')</span>
```

Comment:
I think this embedding of the `$` could be problematic?

Reply:
Yes, those references won't get evaluated correctly. Something like this is needed:

```
          <span class="version">$:_('version <a href="https://github.com/internetarchive/openlibrary/\
commit/%s">%s</a>', git_rev_hash, git_rev_hash)</span>

```